chpl/chpl-service/src/main/java/gov/healthit/chpl/service/CertificationCriterionService.java

```diff
@@ -46,6 +56,88 @@ public CertificationCriterion get(String certificationCriterionDescriptor) {
         }
     }
 
+    public int sortCriteria(CertificationCriterionDTO c1, CertificationCriterionDTO c2) {
+        String valueA = formatCriteriaNumber(c1);
+        String valueB = formatCriteriaNumber(c2);
+        return getCertificationResultSortIndex(valueA) - getCertificationResultSortIndex(valueB);
+    }
+
+    public boolean isCures(CertificationCriterion criterion) {
```

Comment:
Can we rename this function? It would not report d12, d13, b10, or g10 as "cures = true", even though those are also criteria considered Cures. Maybe "hasCuresUpdateTitle" or something

I worry someone trying to filter criteria to those that are actually cures would use this method and get incorrect results without realizing it